designer/client/src/components/toolbars/activities/ActivityPanelRowItem/ActivityItemHeader.tsx

```diff
@@ -227,6 +227,7 @@ const WithOpenVersion = ({
             onClick={() => {
                 changeVersion(scenarioVersion);
             }}
+            title={"Change to " + scenarioVersion + " version"}
```

Comment:
Maybe "Switch to version " + scenarioVersion ?

Reply:
done